src/main/resources/web_en.properties

```diff
@@ -38,32 +36,32 @@ ManagedTask.user = User
 ManagedTaskController.run = Run
 ManagedTaskController.scripting = System Scripting
 ManagedTaskExecution.completed = The execution has been completed.
-ManagedTaskExecution.scheduled = Execution planned, waiting for free thread...
+ManagedTaskExecution.scheduled = Execution planned, waiting for free thread\u2026
 ManagedTaskExecution.started = The execution was started.
 ManagedTaskExecution.waitingForForkedTasks = Wait for started background jobs.
 RythmConfig.jsDateFormat = mm/dd/yyyy
```

Comment:
are those still used?

Reply:
yes these are available as "dateFormat" / "timeformat" in the DefaultGlobalContext